jerry-core/jerry.c

```diff
@@ -119,525 +124,571 @@ jerry_make_api_available (void)
 /**
  * Turn off API availability
  */
-static void
+static inline void __attr_always_inline___
 jerry_make_api_unavailable (void)
 {
   jerry_api_available = false;
 } /* jerry_make_api_unavailable */
 
 /**
- * Returns whether the given jerry_value_t is error.
+ * Construct new TypeError object
+ *
+ * @return TypeError object value
  */
-bool
-jerry_value_is_error (const jerry_value_t value) /**< api value */
+static inline jerry_value_t __attr_always_inline___
+jerry_create_type_error (void)
```

Comment:
I don't think it is worth to inline this, since this is a reare event, and not worth optimizing it for performance.